<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>272</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>461</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomToolBarArea</set>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionHome"/>
   <addaction name="actionSettings"/>
   <addaction name="separator"/>
   <addaction name="actionPower"/>
   <addaction name="separator"/>
   <addaction name="actionActiveCall"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="sources.qrc">
     <normaloff>:/icons/png/settings/Icons/gears.png</normaloff>:/icons/png/settings/Icons/gears.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionPower">
   <property name="icon">
    <iconset resource="sources.qrc">
     <normaloff>:/icons/png/power/Icons/power.png</normaloff>:/icons/png/power/Icons/power.png</iconset>
   </property>
   <property name="text">
    <string>Power</string>
   </property>
  </action>
  <action name="actionHome">
   <property name="icon">
    <iconset resource="sources.qrc">
     <normaloff>:/icons/png/home/Icons/home.png</normaloff>:/icons/png/home/Icons/home.png</iconset>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </action>
  <action name="actionActiveCall">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="sources.qrc">
     <normaloff>:/icons/png/phone/Icons/activeCall.png</normaloff>:/icons/png/phone/Icons/activeCall.png</iconset>
   </property>
   <property name="text">
    <string>activeCall</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="sources.qrc"/>
 </resources>
 <connections/>
</ui>
